lib/pipefy_message/consumer.rb

```diff
@@ -71,10 +71,11 @@ def process_message
 
         logger.info({ message_text: "Calling consumer poller" })
 
-        build_consumer_instance.poller do |payload|
+        build_consumer_instance.poller do |payload, message_attributes|
```

Comment:
Do you think that `metadata` is a better name to be agnostic from a specific AWS term?

Reply:
Done